Tests/OptimizelyTests-APIs/OptimizelyClientTests_OptimizelyJSON_Objc.m

```diff
@@ -63,13 +63,13 @@ - (void)testGetValue {
     // Fetching integer type
     id expectedValue = @1;
     id actualIntValue = @0;
-    XCTAssertTrue([optimizelyJSON getValueWithJsonPath:@"testfield" schema: &actualIntValue]);
+    actualIntValue = [optimizelyJSON getValueWithJsonPath:@"testfield"];
     XCTAssertEqual(expectedValue, actualIntValue);
     
     // Fetching dictionary type
     optimizelyJSON = [[OptimizelyJSON alloc] initWithMap:_map];
     id actualDictionaryValue = @{};
-    XCTAssertTrue([optimizelyJSON getValueWithJsonPath:@"" schema: &actualDictionaryValue]);
+    actualDictionaryValue = [optimizelyJSON getValueWithJsonPath:@""];
     XCTAssertTrue([_map isEqualToDictionary:(NSDictionary *)actualDictionaryValue]);
```

Comment:
Can we add more test cases? Are we allowing "Dictionary" type only? What about user-defined class type?

Reply:
Do we need to convert all swift tests for objc? even though objc is internally calling the swift method? We have covered all cases for swift.

Reply:
I do not think so. We just need show the public APIs work fine. 
We already covered reading primitive data types and map.
 
The test cases we need is:
ObjC custom class (instead of NSDictionary) work with getFeatureVariabaleJSON.

Reply:
ObjectiveC does not support Codable protocol, user will have to get value into an `NSDictionary` and parse it himself which is why i didn't write test related to custom class. I tried writing a test with a custom objc class which implemented NSCodable but swift wasn't able to assigb values into that aswell.